### PR TITLE
Adding global styling for a tags

### PR DESCRIPTION
**Pull request by arnavic**
This PR adds global styling to set the color of any a tags used in Varnish 2.0, adds the link default and contrast colors to the extended palette, and adds a linkContrast class to apply to a tags for contrast styling.

### CODE REVIEW

File: packages/varnish/src/theme/theme.ts [hunk 1]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
...
+            color: ${extended.link.default};
+          }
+          .linkContrast {
+            color: ${extended.link.contrast};
+          }
+        `,
+      },
       MuiLink: {
         styleOverrides: {
           root: ({ ownerState }) => ({
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:62]:
```suggestion
          root: ({ ownerState, theme }) => ({
```

File: packages/varnish/src/theme/theme.ts [hunk 2]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
...
             ...(ownerState.variant === 'dark'
               ? {
-                  color: color2.B2.hex,
+                  color: extended.link.contrast,
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:65]:
```suggestion
                  color: theme.extended.link.contrast,
```

File: packages/varnish/src/theme/theme.ts [hunk 3]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
...
                 }
               : {
-                  color: color2.B3.hex,
+                  color: extended.link.default,
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:68]:
```suggestion
                  color: theme.extended.link.default,
```

File: packages/varnish/src/theme/theme.ts [hunk 4]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
 export const getVarnishDesignTokens = (): ThemeOptions => {
   return {
     components: {
+      MuiCssBaseline: {
+        styleOverrides: () => `
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:51]:
```suggestion
        styleOverrides: (theme) => `
```

File: packages/varnish/src/theme/theme.ts [hunk 5]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
...
+          a {
+            color: ${extended.link.default};
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:53]:
```suggestion
            color: ${theme.extended.link.default};
```

File: packages/varnish/src/theme/theme.ts [hunk 6]
```diff
@@ -47,15 +47,25 @@ declare module '@mui/material/styles' {
...
+          }
+          .linkContrast {
+            color: ${extended.link.contrast};
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:56]:
```suggestion
            color: ${theme.extended.link.contrast};
```

**Review by jonborchardt — COMMENTED** [review]
i think you need to base off the theme, or else they get hardcoded to the default values in the theme file

**Comment by arnavic** [comment]
> i think you need to base off the theme, or else they get hardcoded to the default values in the theme file

makes sense for dynamically changing values, i wasn't sure if it mattered in this case since the paletteExtended values for default and contrast are hard-coded as is but this is a good practice point either way, changed it :)

**Review by jonborchardt — APPROVED** [review]
im assuming it works :-P

**Comment by arnavic** [comment]
> im assuming it works :-P

yes, did test this with different scenarios 👍

**Pull request merged**